lib/matplotlib/_mathtext.py

```diff
@@ -1407,21 +1486,22 @@ def __init__(self, c, width, state, always=False, char_class=Char):
         self.width = char.width
 
 
-def ship(ox, oy, box):
+def ship(box, xy=(0, 0)):
```

Comment:
Doesn't this break the `ship` callers in the existing `MathtextBackend`s?

Reply:
good catch, should be fixed now.